cli.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+const Rapptor = require('rapptor');
+
+const main = async function() {
+  const service = new Rapptor({
+    configPrefix: 'reporter'
```

Comment:
I think you'll need to set the path with __dirname so it'll work when using in other places. Might want to test it out using apptics reports